core-common/src/main/java/org/zalando/nakadi/domain/EventTypeSchemaBase.java

```diff
@@ -17,7 +17,17 @@ public EventTypeSchemaBase(final Type type, final String schema) {
     }
 
     public enum Type {
-        JSON_SCHEMA
+        JSON_SCHEMA("1.0.0"), AVRO_SCHEMA("1");
```

Comment:
Are we sharing anywhere semantic versioning?

Reply:
removed version from here